### ISSUE TITLE
Data binding is required 

### ISSUE THREAD

**Issue by duprass** [refiner-io/mobile-sdk-android#3]
Developers are forced to add databinding = true (even they don't use it) to use library, which is resulted in increasing app size. Otherwise crash on first survey is appearing: 

```
java.lang.NoClassDefFoundError: Failed resolution of: Landroidx/databinding/DataBinderMapperImpl;
at androidx.databinding.DataBindingUtil.<clinit>(DataBindingUtil.java:32)
at androidx.databinding.DataBindingUtil.inflate(DataBindingUtil.java:95)
at io.refiner.ui.base.BaseBottomSheetDialogFragment.onCreateView(SourceFile:1)
at androidx.fragment.app.Fragment.performCreateView(Fragment.java:3114)
at androidx.fragment.app.DialogFragment.performCreateView(DialogFragment.java:775)
at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:557)
at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:272)
at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1943)
at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1845)
at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1782)
at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:3042)
at androidx.fragment.app.FragmentManager.dispatchActivityCreated(FragmentManager.java:2952)
at androidx.fragment.app.FragmentController.dispatchActivityCreated(FragmentController.java:263)
at androidx.fragment.app.FragmentActivity.onStart(FragmentActivity.java:350)
at androidx.appcompat.app.AppCompatActivity.onStart(AppCompatActivity.java:251)
at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1425)
at android.app.Activity.performStart(Activity.java:7825)
at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3294)
at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:221)
at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:201)
at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:173)
at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016)
at android.os.Handler.dispatchMessage(Handler.java:107)
at android.os.Looper.loop(Looper.java:214)
at android.app.ActivityThread.main(ActivityThread.java:7356)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
Caused by: java.lang.ClassNotFoundException: Didn't find class "androidx.databinding.DataBinderMapperImpl" on path: DexPathList[[zip file "/data/app/io.zerion.android.dev-o3Ja424kqfYwogM-Npg7Bg==/base.apk"],nativeLibraryDirectories=[/data/app/io.zerion.android.dev-o3Ja424kqfYwogM-Npg7Bg==/lib/arm64, /data/app/io.zerion.android.dev-o3Ja424kqfYwogM-Npg7Bg==/base.apk!/lib/arm64-v8a, /system/lib64, /system/product/lib64]]
at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:196)
at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
```

**Comment by michal-luszczuk** [comment]
Hello 👋 

I encountered the same problem. 3rd party library like Refiner shouldn't require/force the client to use the `databinding` library.

 @chezmo @tekinalper What can we do to get rid of this requirement?

**Comment by tekinalper** [comment]
We'll be removing databinding requirement in the next version. Fyi.

**Comment by michal-luszczuk** [comment]
> We'll be removing databinding requirement in the next version. Fyi.

Great news! What can you say about the time frame, when can we expect this next version to be live?

**Comment by tekinalper** [comment]
@michal-luszczuk Most probably in a week or 2 weeks at most.

**Comment by tekinalper** [comment]
Resolved in the version 1.3.5
